Library/Formula/libcutl.rb

```diff
@@ -0,0 +1,24 @@
+class Libcutl < Formula
+  homepage "http://www.codesynthesis.com/projects/libcutl/"
+  url "http://www.codesynthesis.com/download/libcutl/1.9/libcutl-1.9.0.tar.gz"
+  sha256 "1b575aa8ed74aa36adc0f755ae9859c6e48166a60779a5564dd21b8cb05afb7d"
+
+  depends_on "gcc" => :build
+
+  def install
+    system "./configure", "--disable-dependency-tracking",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  fails_with :clang do
+    cause <<-EOS.undent
+    Oh! It seems that you only have clang available, or GCC wasn't found!
+    Make sure you GCC is installed and recognized by homebrew.
+    EOS
+  end
+
+  test do
+    system "false"
```

Comment:
>  (which allows me to maintain homebrews packages)

The licensing on the package is GPLv2. Is there some specific reasoning behind you being the [sole](https://github.com/Homebrew/homebrew/pull/37677#issuecomment-78957125) intended submitter? Unless I'm misunderstanding you.

> What should I do?

If the package produces a binary, such as ` bin/example ` then test example. It can be as simple as version or help, but thorough is always better than simple. If it's a library package, write up a small ` .c ` file that links into the library, compile it inside ` test do ` and test that for success.